mdm/common/src/main/java/com/gentics/mesh/core/data/dao/PersistingContentDao.java

```diff
@@ -235,10 +235,11 @@ default void connectFieldContainer(HibNode node, HibNodeFieldContainer newContai
 			// remove existing draft edge
 			if (draftEdge != null) {
 				removeEdge(draftEdge);
-				updateWebrootPathInfo(newContainer, branchUuid, "node_conflicting_segmentfield_update");
 			}
 			// create a new draft edge
 			createContainerEdge(node, newContainer, branchUuid, languageTag, DRAFT);
+
```

Comment:
Because of `updateWebrootPathInfo` being called before creating a new edge, its changes never get indexed upon upload. The upload itself calls `updateWebrootPathInfo` only if the binary field being uploaded is the segment field.